packages/stark-ui/src/modules/breadcrumb/breadcrumb.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from "@angular/core";
+import { BrowserModule } from "@angular/platform-browser";
```

Comment:
BrowserModule should be used only by app.module. 
Could you please use "CommonModule" from "@angular/common" instead ? 😊